pkg/resource/cluster/hook.go

```diff
@@ -84,12 +78,9 @@ func requeueWaitUntilCanModify(r *resource) *ackrequeue.RequeueNeededAfter {
 // has (first, started and then) completed and the cluster reaches an active
 // status.
 func requeueAfterAsyncUpdate() *ackrequeue.RequeueNeededAfter {
-	msg := fmt.Sprintf(
-		"Cluster has started asynchronously updating, cannot be modified until '%s'.",
-		StatusActive,
-	)
 	return ackrequeue.NeededAfter(
-		errors.New(msg),
+		fmt.Errorf("cluster has started asynchronously updating, cannot be modified until '%s'",
+			StatusActive),
```

Comment:
any particular reason for these changes in `hook.go`? I think they are all just style-only changes. Nothing functional, right?

Reply:
Yeah just cleaning things up and keeping it consistent.